### PR TITLE
update for chalk

### PR DESCRIPTION
**Pull request by ghost**
Updated example for chalk

### CODE REVIEW

File: src/feature_app_logging.c [hunk 1]
```diff
@@ -22,15 +27,20 @@ static void select_long_click_release_handler(ClickRecognizerRef recognizer, voi
...
+  window_long_click_subscribe(BUTTON_ID_SELECT, 700, select_long_click_handler, 
+                              select_long_click_release_handler);
 }
 
 static void main_window_load(Window *window) {
   Layer *window_layer = window_get_root_layer(s_main_window);
   GRect bounds = layer_get_frame(window_layer);
-  
+#ifdef PBL_ROUND
+  bounds.origin.y += INSET;
```
Comment (C-D-Lewis) [inline src/feature_app_logging.c]: Prefer using `grect_inset()` instead of manually adjusting the bounds.

File: src/feature_app_logging.c [hunk 2]
```diff
@@ -38,6 +48,9 @@ static void main_window_load(Window *window) {
   text_layer_set_font(s_text_layer, fonts_get_system_font(FONT_KEY_GOTHIC_18_BOLD));
   text_layer_set_text_alignment(s_text_layer, GTextAlignmentCenter);
   layer_add_child(window_layer, text_layer_get_layer(s_text_layer));
+#ifdef PBL_ROUND
+  text_layer_enable_screen_text_flow_and_paging(s_text_layer, INSET);
```
Comment (C-D-Lewis) [inline src/feature_app_logging.c:51]: :+1:

**Comment by C-D-Lewis** [comment]
One adjustment to make, otherwise :+1:

**Comment by ghost** [comment]
@C-D-Lewis Changes have been made. Ready to merge?

**Comment by C-D-Lewis** [comment]
:ship:

**Pull request merged**